com.unity.render-pipelines.universal/ShaderLibrary/UnityInput.hlsl

```diff
@@ -148,6 +148,10 @@ real4 unity_SHBg;
 real4 unity_SHBb;
 real4 unity_SHC;
 
+// Renderer bounding box.
+float4 unity_RendererBounds_Min;
+float4 unity_RendererBounds_Max;
```

Comment:
Do you know the performance impact on adding more per-object data? 
Is there an associated PerObjectData field with this? https://docs.unity3d.com/ScriptReference/Rendering.PerObjectData.html

Reply:
If the property is not used in the shader, then there is no performance impact (because it's removed at the compilation and don't exist in the list of properties tto set in the c++).

Now there are several cases depending on the settings of the project:
- With SRP batcher: no performance hit, this is treated as any other properties and doesn't cause a batch break.
- Without SRP batcher:
- - If instancing is disabled and the property is used in the shader, this will cause a break in the batch for every object that have different bounds (expected as instancing is disabled)
- - Instancing enabled, the property is automatically instanced (like the local to world matrix) and will not cause a batch break.

We decided that having a PerObjectData field for this was useless as it needed to be integrated in ShaderGraph and thus it can be used in any pass of the renderer + the fact that the property is not bound if not used in the shader.